methodoverriding.java

```diff
@@ -0,0 +1,25 @@
+package com.basicsstrong.oop;
+class Vehicle{
```

Comment:
Can you implement another example ?

Reply:
ok I will add it to the branch once i code it

Reply:
I have pushed another example